sdk/python/feast/job.py

```diff
@@ -187,3 +194,98 @@ def to_chunked_dataframe(
 
     def __iter__(self):
         return iter(self.result())
+
+
+class IngestJob:
```

Comment:
Can you implement the __str__ method so that the object can be printed out in a human readable format? See feature_set.py:L87

Reply:
I have added `__str__()` and `__repr__()` to IngestJob to render it in a human readable format.